external/flume/src/test/scala/org/apache/spark/streaming/flume/FlumeStreamSuite.scala

```diff
@@ -76,7 +75,7 @@ class FlumeStreamSuite extends FunSuite with BeforeAndAfter with Matchers with L
 
   /** Find a free port */
   private def findFreePort(): Int = {
-    Utils.startServiceOnPort(23456, (trialPort: Int) => {
+    Utils.startServiceOnPort((math.random * Int.MaxValue).toInt, (trialPort: Int) => {
```

Comment:
`Utils#startServiceOnPort` adjusts the range of port so I simply pass the random value to the method. But I noticed we should pass the value > 0.

      val tryPort = if (startPort == 0) {
        startPort
      } else {
        // If the new port wraps around, do not try a privilege port
        ((startPort + offset - 1024) % (65536 - 1024)) + 1024
      }

So, I'll fix the it.